www/src/pages/en/folder-structure.md

```diff
@@ -20,6 +20,13 @@ Further down, the description of each folder indicates its purpose and if it is
 │  │  ├─ schema.mjs
 │  │  └─ server.mjs
 │  ├─ pages
+│  │  └─ api
+│  │  │  ├─ auth
+│  │  │  │  └─ [...nextauth].ts
+│  │  │  └─ trpc
```

Comment:
Wrong symbol

Reply:
ah yes I see it. The one before "api" line 23, should be sideways T.

Reply:
oh and line 15 before trpc.